### ISSUE TITLE
can't use rbx2-prereq.sh

### ISSUE THREAD

**Issue by lianbo2006** [pirobot/rbx2#1]
When I use `$ sh rbx2-prereq.sh` to install prereq , terminal return a wrong problem:
can't find ros-indigo-moveit-full.
I fix that one to ros-indigo-moveit , and be all right 
Please fix that package name

**Comment by pirobot** [comment]
Are you running this on ROS Indigo or Kinetic?  The package ros-indigo-moveit-full exists in Indigo but not Kinetic.  I have not yet migrated the rbx2 code to Kinetic but I will edit the rbx2_prereq.sh file for the Kinetic branch that is under development.

**Comment by lianbo2006** [comment]
I'm sure my ROS is indigo.
![2017-02-25 09 45 41](https://cloud.githubusercontent.com/assets/22804315/23326951/3d5fdda8-fb3f-11e6-8b78-3639beecd030.png)
This is my terminal,when I want to install(sorry about Chinese)
![qq 20170225094856](https://cloud.githubusercontent.com/assets/22804315/23326981/b8380b36-fb3f-11e6-88b5-61080104669d.png)
When I use `sudo apt-get install ros-indigo-moveit`, it has been installed yesterday.
However, `sudo apt-get install ros-indigo-moveit-full`, it can't find package.
I think maybe is my sources list courses this problem, I use http://mirrors.ustc.edu.cn/ubuntu/.
Thanks for your reply a lot.

**Comment by pirobot** [comment]
OK, that's odd.  What version of Ubuntu are you running?

**Comment by lianbo2006** [comment]
Ubuntu 14.04 LTS

**Comment by pirobot** [comment]
It looks like ros-indigo-moveit-full got left out of the Ubuntu debs for awhile and I see a note on the ros-users list that it has been put back in.  What happens if you run:

```
sudo apt-get update
sudo apt-get install ros-indigo-moveit-full
```

**Comment by lianbo2006** [comment]
Thanks for reply.
I intall moveit-full after update, and there is really a new package been installed. However, I remembered I also did update last time. Well, the problem has been solved.
![qq 20170302083707](https://cloud.githubusercontent.com/assets/22804315/23487806/7804e8ee-ff23-11e6-839e-a2f8c7366284.png)
![qq 20170302082540](https://cloud.githubusercontent.com/assets/22804315/23487572/efccc498-ff21-11e6-9768-663f773a9a1e.png)
